1.10/networking/virtual-networks/index.md

```diff
@@ -5,73 +5,60 @@ feature_maturity: preview
-
-* Certain names are reserved and cannot be used as DC/OS Overlay names. The is because DC/OS Overlay uses Docker networking underneath to connect Docker containers to the overlay, which in turn reserves certain network names. The reserved names are: `host`, `bridge` and `default`.
-
-* [Marathon health checks](/docs/1.10/deploying-services/creating-services/health-checks/) will not work with certain DC/OS Overlay configurations. If you are not using the default DC/OS Overlay configuration and Marathon is isolated from the virtual network, health checks will fail consistently even if the service is healthy.
-
-  Marathon health checks _will_ work in any of the following circumstances:
-
-  * You are using the default DC/OS Overlay configuration.
-  * Marathon has access to the virtual network.
-  * You use a [`command` health check](/docs/1.10/deploying-services/creating-services/health-checks/).
-
-# Virtual Network Service: DNS
-
-The [Virtual Network Service](/docs/1.10/overview/architecture/components/) maps names to IPs on your virtual network. You can use these DNS addresses to access your task:
-
-* **Container IP:** Provides the container IP address: `<taskname>.<framework_name>.containerip.dcos.thisdcos.directory`
-* **Auto IP:** Provides a best guess of a task's IP address: `<taskname>.<framework_name>.autoip.dcos.thisdcos.directory`. This is used during migrations to the overlay.
-
-Terminology:
-* `taskname`: The name of the task
-* `framework_name`: The name of the framework, if you are unsure, it is likely `marathon`
+**Note:** Isolation guarantees among subnets depend on your CNI implementation and/or your firewall policies.
+
+# Using Virtual Networks
+
+First, you or the data center operator needs to [configure the virtual networks](/docs/1.10/networking/virtual-networks/).
+
+Virtual networks are configured at install time. You or the data center operator will specify a canonical name for each network in the `config.yaml`. When your service needs to launch a container, refer to it by that canonical name.
+
+To use an virtual network in a Marathon app definition, specify the `"network": "USER"` property along with an `ipAddress` field in the form: `{"ipAddress": {"network": "$MYNETWORK"}}`. The value of `$MYNETWORK` is the canonical name of the network.
+
+# Example
+
+The following Marathon application definition specifies a network named `dcos-1`, which refers to the target DC/OS virtual network of the same name.
+
+```json
+{
+   "id":"my-networking",
+   "cmd":"env; ip -o addr; sleep 30",
+   "cpus":0.10,
+   "mem":64,
+   "instances":1,
+   "backoffFactor":1.14472988585,
+   "backoffSeconds":5,
+   "ipAddress":{
+      "networkName":"dcos-1"
+   },
+   "container":{
+      "type":"DOCKER",
+      "docker":{
+         "network":"USER",
```

Comment:
The **network** property is now **networks** and is at the app definition level, not under docker.